### PR TITLE
Added command line parser and code to support migration

### LINKED ISSUE #85: Refactor Face recognition app
Following changes are required:
- [x] Remove "Sapphire" key from code
- [x] Add command line parser
- [x] Use AtLeastOnceRPCPolicy DM
- [x] Add code changes to make Kernel server run with App.
- [x] Update Readme file for 
      - opencv installation
      - Add comments in code

### PR DESCRIPTION
**Pull request by AmitRoushan**
This PR has following changes
1. Removed "sapphire"  key.
2. Added command line parser for app.
3. Updated App code to make it run along side kernel server
4. Added "AtLeastOnceRPCPolicy" for Recognition micro service.
5. Updated Readme.md file
6. Updated video and image file

Fixes #85

### CODE REVIEW

File: face-recognition-demo/src/main/java/application/DemoAppStart.java [hunk 1]
```diff
@@ -1,99 +1,135 @@
...
-        if ( sourceType.equalsIgnoreCase("camera") || sourceType.equalsIgnoreCase("video") ) {
+        if (sourceType.equalsIgnoreCase("camera") || sourceType.equalsIgnoreCase("video")) {
             frameGenerator = new FrameGenerator(sourceType);
         } else {
             logger.severe("Incorrect source specified, use either \"camera\" or \"video\"");
             return;
         }
 
-        try {
+        if (appArgs.inferenceType.equalsIgnoreCase("detection")) { // run face detection locally
```
Comment (VenuReddyHuawei) [inline face-recognition-demo/src/main/java/application/DemoAppStart.java]: Redundant comment - `// run face detection locally`

Comment (AmitRoushan) [reply face-recognition-demo/src/main/java/application/DemoAppStart.java]: removed comment

**Comment by VenuReddyHuawei** [comment]
LGTM

**Comment by Vishwa4jeet** [comment]
@VenuReddyHuawei  is merging it.

**Pull request merged**